pkg/server/api/svid/v1/server.go

```diff
@@ -58,21 +58,12 @@ func (s server) MintX509SVID(ctx context.Context, req *svid.MintX509SVIDRequest)
 func (s server) MintJWTSVID(ctx context.Context, req *svid.MintJWTSVIDRequest) (*svid.MintJWTSVIDResponse, error) {
 	log := rpccontext.Logger(ctx)
 
-	if req.Id == nil {
-		log.Error("Request must specify SPIFFE ID")
-		return nil, status.Error(codes.InvalidArgument, "request must specify SPIFFE ID")
-	}
-	spiffeID, err := spiffeid.New(req.Id.TrustDomain, req.Id.Path)
+	spiffeID, err := api.IDFromProto(req.Id)
```

Comment:
Hmm, one thing I missed on the MintX509SVID review is that the current Mint methods on the registration API only allow workload SVIDs (i.e. non-empty path component not in the reserved `/spire/*` namespace). I think we should mimic that behavior.